flink-end-to-end-tests/flink-stream-sql-test/src/main/java/org/apache/flink/sql/tests/StreamSQLTestProgram.java

```diff
@@ -202,7 +228,11 @@ public GeneratorTableSource(int numKeys, float recordsPerKeyAndSecond, int durat
 
 		@Override
 		public DataStream<Row> getDataStream(StreamExecutionEnvironment execEnv) {
-			return execEnv.addSource(new Generator(numKeys, recordsPerKeyAndSecond, durationSeconds, offsetSeconds));
+			return execEnv
+				.addSource(new Generator(numKeys, recordsPerKeyAndSecond, durationSeconds, offsetSeconds))
```

Comment:
What is the long term plan to fix it? It is quite a substantial change to the test.

Reply:
FLINK-13494 will fix the parallelism setting logic for source/sink, and remain compliant to Old planner. Ofter that, this explicit setting could be removed.

Reply:
FLINK-13494 will fix the parallelism setting logic for source/sink, and remain compliant to Old planner. Ofter that, this explicit setting could be removed.

Reply:
FLINK-13494 will fix the parallelism setting logic for source/sink, and remain compliant to Old planner. Ofter that, this explicit setting could be removed.

Reply:
This change should be reverted in #9277.

Reply:
Personally I would prefer to get the #9277 in first.

Reply:
I'm fine with that.

Reply:
#9277 is merged. We can remove this change and see if it works now.

Reply:
Yes, once this change is reverted and travis passes I would say we can merge this PR.